apps/staking/app/register/[nodeId]/NodeRegistrationForm.tsx

```diff
@@ -0,0 +1,516 @@
+              watchedStakeAmount={watchedStakeAmount}
+              onChange={onChange}
+              field={field}
+              value={value}
+            />
+          )}
+        />
+        <Accordion type="single" collapsible>
+          <AccordionItem value="item-1">
+            <AccordionTrigger>{dictionary('advancedOptions.title')}</AccordionTrigger>
+            <AccordionContent className="flex flex-col gap-2">
+              {!isSolo ? (
+                <ActionModuleRow
+                  label={dictionary('advancedOptions.autoRegister')}
+                  tooltip={dictionary('advancedOptions.autoRegisterDescription')}
+                >
+                  <FormItem className="flex flex-col">
+                    <FormField
+                      control={form.control}
+                      name="autoRegister"
+                      render={({ field: { value, onChange } }) => (
+                        <Switch
+                          checked={value}
+                          onCheckedChange={onChange}
+                          disabled={disableInputs}
+                        />
+                      )}
+                    />
+                    <FormMessage />
+                  </FormItem>
+                </ActionModuleRow>
+              ) : null}
+              <FormItem className="flex flex-col">
+                <FormField
+                  control={form.control}
+                  name="beneficiaryAddress"
+                  render={({ field }) => (
+                    <div className="flex w-full flex-col items-center gap-2">
+                      <div className="flex w-full justify-between gap-2 text-nowrap">
+                        <span className="inline-flex items-center gap-2 text-nowrap align-middle">
+                          {dictionary('advancedOptions.beneficiary')}
+                          <ActionModuleTooltip>
+                            {dictionary('advancedOptions.beneficiaryDescription')}
+                          </ActionModuleTooltip>
+                        </span>
+                      </div>
+                      <Input
+                        placeholder={'0x...'}
+                        disabled={!isConnected || disableInputs}
+                        className="w-full"
+                        {...field}
+                      />
+                    </div>
+                  )}
+                />
+                <FormMessage />
+              </FormItem>
+              {!isSolo ? (
+                !isReservedRegDisabled ? (
+                  <Button
+                    data-testid={ButtonDataTestId.Register_Add_Reserved}
+                    rounded="lg"
+                    size="lg"
+                    variant="outline"
+                    disabled={
+                      true ||
+                      isRegistrationPausedFlagEnabled ||
+                      isRemoteFlagLoading ||
+                      !form.formState.isValid
+                    }
```

Comment:
Leave a comment saying this feature isn't ready yet.

Reply:
According to yourself the true can be removed

Reply:
added comment, keeping true